### ISSUE TITLE
Unknown Dialect when query against and undefined explore

### ISSUE THREAD

**Issue by lloydtabb** [looker-open-source/malloy#157]
Test in nomodel.spec.malloy

```
        explore: foo is table('malloytest.state_facts'){primary_key: state}
        query: foox->{aggregate: c is count()}
```

**Comment by mtoy-googly-moogly** [comment]
Smallest model for this is just `query: x->{ group_by: y }`

**Comment by mtoy-googly-moogly** [comment]
Fixed 75e06687815d4655f3da79827818c92e71758d38
